src/main.cpp

```diff
@@ -314,6 +318,7 @@ void getConfigSchemaJson(JsonVariant json)
   JsonObject fadeIntervalUs = properties.createNestedObject("fadeIntervalUs");
   fadeIntervalUs["type"] = "integer";
   fadeIntervalUs["minimum"] = 0;
+  fadeIntervalUs["description"] = "The rate for fade to work in Millis - default is 500ms";
```

Comment:
`Time to fade from off to on, or vice versa, in milliseconds (default is 500ms)`